inc/connector/namespace.php

```diff
@@ -295,14 +293,13 @@ function pre_get_scheduled_event( $pre, $hook, $args, $timestamp ) {
 		return false;
 	}
 
-	$result = $job[0];
+	$result = reset( $job );
 
 	$value = (object) [
 		'hook'      => $result->hook,
 		'timestamp' => $result->nextrun,
 		'schedule'  => $result->schedule,
 		'args'      => $result->args,
-		'_job'      => $result,
```

Comment:
This was causing a core test to fail and changed the existing behaviour of Cavalcade.

Reply:
I saw that but was looking through and in some instances this value is used to get a job instance as far as I could tell. I may be mistaken here but I'd like to be absolutely sure.

When you say it changed the existing behaviour do you mean Cavalcade never used to return this?

Reply:
Derp, I see. Core does some post processing on the results from the option value 👍

Reply:
Yeah, that's right.

It may be legit to return the `_job` though if you'd prefer but let's add a test similar to Core's if you decide it's needed.

Reply:
Let's roll with this for now :) It's still possible to get the job instance easily enough.